src/components/Pages/PageSearch.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import MovieListConnect from '../Molecules/MovieList';
+import { setMovies } from '../../redux/actions';
+import './GalleryStyles.scss';
+import TemplateDefault from '../Templates/TemplateDefault';
+import LoadingAnim from '../Atoms/LoadingAnim';
+
+const PageSearch = () => {
+  const dispatch = useDispatch();
+  const { id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [message, setMessage] = useState();
+
+  useEffect(() => {
+    const getMovies = async () => {
```

Comment:
This looks almost identical to the same code seen in `PageGallery.jsx`. It would be worth consolidating these into a single method for ease of maintenance.